test/models/site_test.rb

```diff
@@ -20,4 +26,32 @@ class SiteTest < ActiveSupport::TestCase
     assert_equal(0, Post.count)
     assert_equal(0, Image.count)
   end
+
+  sub_test_case 'name' do
+    test 'null' do
+      site = Site.create(js_url: "http://example.com/application.js",
+                         css_url: "http://example.com/application.css")
+      assert_false(site.valid?)
+      assert_equal("can't be blank", site.errors[:name][0])
```

Comment:
If site has some other errors, this test may failed :broken_heart: 
Can you use other assertion what something like [`assert_includes`](http://docs.ruby-lang.org/ja/2.1.0/class/MiniTest=3a=3aAssertions.html#I_ASSERT_INCLUDES)?